Cargo.toml

```diff
@@ -58,6 +58,7 @@ nightly = ["num-bigint/nightly"]
 serde = ["num-bigint/serde", "serde_crate"]
 expose-internals = []
 std = ["digest/std", "pkcs1/std", "pkcs8/std", "rand_core/std", "signature/std"]
+digest-preview = ["signature/digest-preview" ]
```

Comment:
The `rsa` crate already has a hard dependency on `digest`, so you can just enable this unilaterally:

```toml
signature = { version = ">=1.5, <1.7", default-features = false , features = ["digest-preview", "rand-preview"] }
```